build.gradle

```diff
@@ -63,6 +63,8 @@ dependencyManagement {
     }
 }
 
+ext ['spring-security.version'] = '5.3.0.RELEASE' // pinned due to CVE-2018-1258 in 5.2.2.RELEASE brought in by spring-boot-starter-oauth2-resource-server-2.2.5.RELEASE
```

Comment:
don't think 5.3.0 fixes CVE-2018-1258 though?  You've still got `dependency-check-suppress-spring.xml` suppressing it

Reply:
not that I think it is a bad idea upgrading it